### PR TITLE
ROCm3.5 miopen recipe

### PR DESCRIPTION
**Pull request by gkathirv**

### CODE REVIEW

File: var/spack/repos/builtin/packages/miopen-hip/package.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    """AMD's library for high performance machine learning primitives."""
+
+    homepage = "https://github.com/ROCmSoftwarePlatform/MIOpen"
+    url = "https://github.com/ROCmSoftwarePlatform/MIOpen/archive/rocm-3.5.0.tar.gz"
+
+    maintainers = ['srekolam', 'arjun-raj-kuppala']
+
+    version('3.5.0', sha256='aa362e69c4dce7f5751f0ee04c745735ea5454c8101050e9b92cc60fa3c0fb82')
+
+    variant('build_type', default='Release', description='CMake build type')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/miopen-hip/package.py]: What possible values does this have?

Comment (gkathirv) [reply var/spack/repos/builtin/packages/miopen-hip/package.py]: > What possible values does this have?

Release & debug variants available, I have added the same.

**Review by adamjstewart — APPROVED** [review]

**Comment by adamjstewart** [comment]
@srekolam @arjun-raj-kuppala can you review this PR? I'll merge when you say it's good to go.

**Comment by srekolam** [comment]
@adamjstewart , yeah , I am okay with the changes.

**Pull request merged**